tests/unittest/_torch/sampler/test_torch_sampler.py

```
@@ -635,231 +635,259 @@ def _uut(res=res):
-            for actual, request in zip(reasons, requests, strict=True):
-                expected = request.finish_reasons
-                msg = f"actual={[FinishReason(reason) for reason in actual]} != expected={expected}\nFor {request}"
-                assert actual == [reason.value for reason in expected], msg
+        _run_test_with_warmup(uut_provider)
 
-        return run, sampler
+    @classmethod
+    def test_are_stop_words_isnt_called_when_no_stop_words(cls, monkeypatch: pytest.MonkeyPatch):
+        """We don't want to call are_stop_words when there are no stop words because it's expensive"""
 
+        def stop_words_that_raises(*args, **kwargs):
```

Ruff (0.15.0)

[warning] Unused function argument: `args`

(ARG001)